src/plugins/shared/pluginResponses.ts

```diff
@@ -11,4 +11,13 @@ export const getAKSClusterPluginResponse = (): GitHubCopilotForAzureChatPluginRe
         commandID: "aks.aksCreateClusterFromCopilot",
         buttonLabel: "Get started"
     }
+}
+
+//Deploy manifest to AKS cluster plugin
```

Comment:
Copying comment over from https://github.com/Azure/vscode-aks-tools/pull/966#discussion_r1790934232: 

deploying an application to AKS requires few more steps, I agree this is all for POC purposes

App image should be deployed beforehand to container registry, can you include those as well ?
if it is ACR (Azure Container Registry), acrpull role assignment should be added for the user to deploy to AKS successfully.